config/sidekiq.yml

```diff
@@ -4,3 +4,6 @@
   expired_bet_worker:
     every: '1h'
     class: ExpiredBetWorker
+  delete_old_bets_worker:
+    cron: '0 0 1 * * Europe/London'
```

Comment:
Is the intent for this to run only on the 1st of each month?

Reply:
Yes